### DISCUSSION TITLE
Pansharpening LandSat8

### DISCUSSION THREAD

**Discussion by FelipeSBarros** [semiautomaticgit/SemiAutomaticClassificationPlugin#261] [General]
Dear @semiautomaticgit 
sorry for bothering again.
I am working on a Landsat 8 images.
When running the preprocessing tools, I have set to run the DOS correction and pansharpening.
As expected, I got only three bands output with prefix "PAN_". And all others as "RT_".
But looking to those that I was expecting only atmospheric correction ("RT_"), I have then with 15, -15 pixel size;

![image](https://user-images.githubusercontent.com/7208674/233646793-4a764a36-fda3-4d0a-82e8-dc5b6ca00ef8.png)

Did I misunderstood something?
Thanks in advance

**Comment by semiautomaticgit** [comment]
Hello @FelipeSBarros ,
all the bands have the same spatial resolution in order to potentially use all the bands in a classification. 
Of course it is only a resampling, and you could ignore those bands if not required.

**Reply by FelipeSBarros** [reply]
Thanks for the clarification, @semiautomaticgit !
So, if I understand right, when running the preprocessing tool with applying the pansharpening, those bands that are not in the range of the pan sharpening septal resolution will be resampled so they could be used together with those that had been fusioned.
Is that right?

I probably have missed something, but don’t you think worth having a check box for specifying wether or not the resampling should be done?
Best regards

**Comment by semiautomaticgit** [comment]
You are very welcome!

> So, if I understand right, when running the preprocessing tool with applying the pansharpening, those bands that are not in the range of the pan sharpening septal resolution will be resampled so they could be used together with those that had been fusioned.

It is right.

Yes, it could also have a checkbox. I'll think about adding this option.

Best regards

**Comment by FelipeSBarros** [comment]
Hi @semiautomaticgit ! Saw that you closed this discussion.
Have you adressed the point I raised in the new version? Or it still resamplig the bands to the same spatial resolution as the panchromatic band?
Best regards

Felipe

**Reply by semiautomaticgit** [reply]
Hi @FelipeSBarros , I actually closed it because pansharpening is not available in the new version 8.
Hopefully it will be added in the future.

**Reply by FelipeSBarros** [reply]
Ah, perfect. Thanks to mention. I didn't had time to test the newest version. 
Congratulations and thanks for the work done!

**Reply by semiautomaticgit** [reply]
You are very welcome! And thank you for your support!
